docs/sources/setup/install/helm/monitor-and-alert/with-local-monitoring.md

```diff
@@ -11,203 +11,163 @@ keywords:
   - alerting
 ---
 
-# Configure monitoring and alerting
+# Monitor Loki using a local LGTM stack
 
-By default this Helm Chart configures meta-monitoring of metrics (service monitoring) and logs (self monitoring). This topic will walk you through configuring monitoring using a monitoring solution local to the same cluster where Loki is installed.
+This topic will walk you through deploying the local meta-monitoring helm to monitor a Loki installation that is installed with the Helm chart. This approach leverages many of the chart's _self monitoring_ features, but instead of sending logs back to Loki itself, it sends them to a small LGTM stack running within the meta namespace. 
```

Comment:
```suggestion
This topic will walk you through using the meta-monitoring Helm chart to deploy a local stack to monitor your production Loki installation. This approach leverages many of the chart's _self monitoring_ features, but instead of sending logs back to Loki itself, it sends them to a small Loki, Grafana, Tempo, Mimir (LGTM) stack running within the `meta` namespace. 
```